subjects/printif/README.md

```diff
@@ -0,0 +1,45 @@
+## printif
+
+### Instructions
+
+Write a function that takes a `string` as an argument and returns the letter `G` if the argument length is more than 3, otherwise returns **Invalid Input\n**.
```

Comment:
- Instruction fix: "...argument length is bigger than 3, otherwise returns `Invalid Input\n`." 
- Also missing a space in between line 5 and line 6.

Reply:
fixed